### PR TITLE
Added run-and-connect file

### PR DESCRIPTION
**Pull request by jefurbee**
**Summary**
I've moved the creating profiles, connecting to SAS, and running code sections into a new file connect-and-run.md.

**Testing**
How did you test this change?

### CODE REVIEW

File: connect-and-run.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+**Note:** Your sign in status will persist in VS Code. You can view it and sign out from VS Code's `Accounts` menu.
+
+3. VS Code connects to SAS and runs the code.
+4. The results are displayed in the application.
+5. The SAS output log and error information are displayed in the applicaiton.
+
+<img src="doc/images/runCode2.png"/>
+
+**Note:** Your sign in status will persist in VS Code. You can view it and sign out from VS Code's `Accounts` menu.
```
Comment (scnwwu) [inline connect-and-run.md:193]: duplicated with line 185 above

File: README.md [hunk 2]
```diff
@@ -143,105 +143,16 @@ The following commands are supported for profiles:
 | `SAS.updateProfile` | SAS: Update Connection profile         |
```
Comment (scnwwu) [inline README.md:143]: looks like the connect-and-run.md also includes this table, so just wondering do we still want this table here?

File: connect-and-run.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    "ssh_test": {
+        "connectionType": "ssh",  
+        "host": "host.machine.name",  
+        "saspath": "/path/to/sas/executable",  
+        "username": "username",  
+        "port": 22
+    }
+```
+Note: the default path to the SAS executable (saspath) is /opt/sasinside/SASHome/SASFoundation/9.4/bin/sas_u8. Check with your SAS administrator for the exact path.  
+7. Add the public part of the keypair to the SAS server. Add the contents of the key file to the ~/.ssh/authorized_keys file.
```
Comment (scnwwu) [inline connect-and-run.md:88]: looks like should be 9.

File: connect-and-run.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+   
+6. Define an entry in ~/.ssh/config of the form:
+```
+    Host host.machine.name 
+        AddKeysToAgent yes
+        IdentityFile /path/to/private/key/with/passphrase 
+```
+Note: if ~/.ssh/config does not exist, run the following Powershell command to create it: `Out-File -FilePath config`  
+7. Add the private key to ssh-agent: ssh-add /path/to/private/key/with/passphrase
+8. In VS Code, define a connection profile  (see detailed instructions below in the [Add New SAS 9.4 Remote Profile](#add-new-sas-94-remote-profile) section). The connection for the remote server is stored in the settings.json file.
```
Comment (scnwwu) [inline connect-and-run.md:77]: needs a blank line to be displayed in a new line

File: README.md [hunk 5]
```diff
@@ -143,105 +143,16 @@ The following commands are supported for profiles:
...
-2. For a secure connection to SAS Viya we use an authorization code for authentication. Complete these steps to connect.
-
-   1.1. VS Code may prompt you that the extension wants to sign in. Click 'Allow'.
-
-   1.2. VS Code may prompt you about opening an external web site. Click 'Open'.
-
-   1.3. This will take you to a SAS Logon prompt. Log in with your SAS credentials.
-
-   1.4. SAS returns an authorization code. Copy this code.
+Details on creating and managing profiles is available on the [Connect and Run page](connect-and-run.md).
```
Comment (scnwwu) [inline README.md:153]: perhaps should say Details on running code here

**Review by scnwwu — COMMENTED** [review]
Please run `npm run format` to pass the build. Thanks.
I think the original runCode.png can be deleted if not needed.

File: connect-and-run.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+**Note**: An attempt to create the varible using Powershell command line did not register; suggest using these GUI instructions.
+
+3. Ensure ssh-agent service is running and set startup type to automatic; commands found in [this link](https://dev.to/aka_anoop/how-to-enable-openssh-agent-to-access-your-github-repositories-on-windows-powershell-1ab8)  
+
+4. [Generate ed25519 keys](https://medium.com/risan/upgrade-your-ssh-key-to-ed25519-c6e8d60d3c54) with the following command (email address is not binding; use any):  
+
+```
+     ssh-keygen -o -a 100 -t ed25519 -f ~/.ssh/id_ed25519 -C "youremail@company.com"
+```
+5. You’ll be asked a series of questions. First, if you didn not provide a path, a default one is provided. Also, if you wish to add a passphrase enter it. Pressing the ‘Enter’ key for each question accepts the default key name and does not password protect your key.
```
Comment (smorrisj) [inline connect-and-run.md:62]: typo

**Review by smorrisj — COMMENTED** [review]
Overall looks pretty good. I thought that we talked about moving some of the lower level details about connection profile setup to the wiki pages? The reason being that our documentation could evolve independently of source and be able to change outside of releases. Wanted to see if we're still thinking about this or?

File: README.md (hunk 2 above)
Comment (jefurbee) [reply README.md:143]: I think I'd like to keep it in both places for reference.

File: connect-and-run.md (hunk 1 above)
Comment (jefurbee) [reply connect-and-run.md:193]: Removing the line 185 version of the text.

File: connect-and-run.md (hunk 3 above)
Comment (jefurbee) [reply connect-and-run.md:88]: Fixed items 7,8, and 9 so they now appear correctly, in order, on new lines.

File: connect-and-run.md (hunk 4 above)
Comment (jefurbee) [reply connect-and-run.md:77]: Add two spaces at the end of number 7. Now number 8 appears on a separate line.

File: README.md (hunk 5 above)
Comment (jefurbee) [reply README.md:153]: Fixed to read 'Details on running code is available......'

**Review by scnwwu — APPROVED** [review]

**Pull request merged**